spec/services/course/assessment/answer/auto_grading_service_spec.rb

```diff
@@ -3,23 +3,22 @@
 RSpec.describe Course::Assessment::Answer::AutoGradingService do
   let(:instance) { create(:instance) }
   with_tenant(:instance) do
-    let(:answer) { build(:course_assessment_answer_multiple_response, :submitted).answer }
-    let(:auto_grading) { build(:course_assessment_answer_auto_grading, answer: answer) }
+    let(:answer) { create(:course_assessment_answer_multiple_response, :submitted).answer }
+    let(:auto_grading) { create(:course_assessment_answer_auto_grading, answer: answer) }
 
     describe '.grade' do
       subject { Course::Assessment::Answer::AutoGradingService.grade(auto_grading) }
 
-      it 'grades the grading' do
+      it 'grades the answer' do
         subject
 
-        expect(auto_grading).to be_graded
+        expect(auto_grading.answer).to be_graded
       end
     end
 
     describe '#grade' do
-      it 'sets the status to graded' do
+      it "sets the answer's status to graded" do
```

Comment:
We prefer `"sets the answer's status to graded"` or `'sets the answer\'s status to graded'` ? I remember once you mentioned to use second form.